### PR TITLE
Adding 🚇  emoji for infrastructure

### LINKED ISSUE #33: Add Emoji for Infrastructure work
First off: love the idea/project. I am currently implementing it for [a project of mine], and even though it is with 4 contributors still quite small, already found myself with a few rather odd places with the emoji. I will post them as separate issue so we can discuss each suggestion I have.

This one is about "infrastructure work" someone might contributed. In my case this means setting up build-services (like travis), hosting, maintain servers etc. I suggest to use the "power plug" emoji: 🔌

(One could argue that this could be merged with "tools" but then we should adapt the language for that one to explicitly include that sort of thing)

### PR DESCRIPTION
**Pull request by ligthyear**
fixes #33

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,10 +63,10 @@ Emoji | Represents | Links to
...
.../commits?author=bogas04) 👀 | [![Ben Briggs](https://avatars1.githubusercontent.com/u/1282980?v=3&s=100)<br /><sub>Ben Briggs</sub>](http://beneb.info)<br />[📖](https://github.com/kentcdodds/all-contributors/commits?author=ben-eb) 👀 | [![James Monger](https://avatars1.githubusercontent.com/u/2037007?v=3&s=100)<br /><sub>James Monger</sub>](http://github.com/Jameskmonger)<br />[📖](https://github.com/kentcdodds/all-contributors/commits?author=Jameskmonger) | [![Jeroen Engels](https://avatars.githubusercontent.com/u/3869412?v=3&s=100)<br /><sub>Jeroen Engels</sub>](https://github.com/jfmengels)<br />[📖](https://github.com/kentcdodds/all-contributors/commits?author=jfmengels) [🔧](https://www.npmjs.com/package/all-contributors-cli) 👀 | [![Chris Simpkins](https://avatars0.githubusercontent.com/u/4249591?v=3&s=100)<br /><sub>Chris Simpkins</sub>](http://github.com/chrissimpkins)<br />[📖](https://github.com/kentcdodds/all-contributors/commits?author=chrissimpkins) 👀 | [![Frederic Hemberger](https://avatars0.githubusercontent.com/u/153481?v=3&s=100)<br /><sub>F. Hemberger</sub>](http://github.com/fhemberger)<br />[📖](https://github.com/kentcdodds/all-contributors/commits?author=fhemberger) |
```
Comment (kentcdodds) [inline README.md]: What changes did you make to these lines? I can't see what the difference is :-/

Comment (ligthyear) [inline README.md]: none. let me remove that from the changeset.

**Comment by ligthyear** [comment]
Rebased. Now only adding that line..

**Comment by kentcdodds** [comment]
Looks good to me! Who wants to merge this?

**Comment by ben-eb** [comment]
Thanks! :tada:

**Comment by kentcdodds** [comment]
celebrate-DKnMqdm9i980E

**Pull request merged**